snoop/data/filesystem.py

```diff
@@ -73,34 +73,36 @@ def handle_file(file_pk):
     file = models.File.objects.get(pk=file_pk)
     depends_on = {}
 
-    if archives.is_archive(file.blob.mime_type):
-        digest_blob = file.blob
-        unarchive_task = archives.unarchive.laterz(file.blob)
+    if archives.is_archive(file.original.mime_type):
+        file.blob = file.original
+        unarchive_task = archives.unarchive.laterz(file.original)
```

Comment:
the `unarchive` operation should happen on the mangled `File.blob`, not the original